src/presentation_exchange.rs

```diff
@@ -277,6 +319,71 @@ impl PresentationDefinition {
         self
     }
 
+    /// Validate a presentation submission against the presentation definition.
+    ///
+    /// Checks the underlying presentation submission parsed from the authorization response,
+    /// against the input descriptors of the presentation definition.
+    pub async fn validate_authorization_response(
+        &self,
+        auth_response: &AuthorizationResponse,
+    ) -> Result<()> {
+        match auth_response {
+            AuthorizationResponse::Jwt(_jwt) => {
+                // TODO: Handle JWT Encoded authorization response.
+
+                bail!("Authorization Response Presentation Definition Validation Not Implemented.")
+            }
+            AuthorizationResponse::Unencoded(response) => {
+                let presentation_submission = response.presentation_submission().parsed();
+
+                let jwt = response.vp_token().0.clone();
+
+                // TODO: Verify the JWT.
+                // let jws = CompactJWSString::from_string(jwt.clone()).context("Invalid JWT.")?;
+                // let resolver: VerificationMethodDIDResolver<DIDKey, AnyJwkMethod> =
+                //     VerificationMethodDIDResolver::new(DIDKey);
+                // let params = VerificationParameters::from_resolver(resolver);
+
+                // if let Err(e) = jws.verify(params).await {
+                //     bail!("JWT Verification Failed: {:?}", e)
+                // }
```

Comment:
TODO: Verify the JWT

Reply:
commit [1398cad](https://github.com/spruceid/oid4vp-rs/pull/8/commits/1398cadffd13c2a03fc69fc8514656a7830c3f73) resolves